QuizBowlDiscordScoreTracker/Bot.cs

```diff
@@ -193,6 +198,8 @@ private async Task PromptNextPlayer(GameState state, ITextChannel textChannel)
 
                 IGuildUser user = await textChannel.Guild.GetUserAsync(userId);
                 await textChannel.SendMessageAsync(user.Mention);
+                await hubContext.Clients.Group(textChannel.Id.ToString(CultureInfo.InvariantCulture))
```

Comment:
Consider making a method to get the groupName that returns

`textChannel.Id.ToString(CultureInfo.InvariantCulture)`

Then any place that needs a group (the two cases here, or more in the future) just need to call one method to always get the correct value.

On a side note, if we ever change the site so readers can perform actions on it (clear, scoring, etc.), then we'll need to make the connectionId secret. We could either change the groupName or have a mapping from connection ID to groupName. This method could help there, too.

Reply:
implemented in 4823a0e

wasn't sure where to put this, so I tossed it in a static method on `Bot`. Definitely could be moved to a Util class in Web/ or something if that's preferable

Reply:
Static method is fine.